Erreur 500 sur la page de login

## Comportement attendu

J’accède à https://permacoop.fairness.coop sur mobile, j’arrive sur la page de connexion, prêt à me connecter

## Comportement réel

J’obtiens une page erreur 500

## Reproduire

https://permacoop.fairness.coop

Se déconnecter si besoin

## Piste de résolution

Voir les logs ? Ça semble lié à la migration SvelteKit… Peut être une bonne idée d’ajouter des tests E2E mobiles.

Edit : en fait ça se passe sur desktop aussi !

Erreur côté client SvelteKit:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/root/permacoop/client/kit/build/server/chunks/1-c8662b77.js' imported from /root/permacoop/client/kit/build/server/manifest.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:330:11)
    at moduleResolve (node:internal/modules/esm/resolve:907:10)
    at defaultResolve (node:internal/modules/esm/resolve:1115:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:837:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ESMLoader.import (node:internal/modules/esm/loader:521:22)
    at importModuleDynamically (node:internal/modules/esm/translators:110:35)
    at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
```

J'avais résolu en faisant un build manuel sur la VM, mais suite au redéploiement c'est de retour...

--- a/ansible/roles/web/templates/.env-server.j2
+++ b/ansible/roles/web/templates/.env-server.j2
@@ -1,5 +1,5 @@
 DATABASE_HOST="{{ db_host }}"
-DATABASE_PORT="{{ db_password }}"
+DATABASE_PORT="{{ db_port }}"
 DATABASE_USERNAME="{{ db_username }}"
 DATABASE_PASSWORD="{{ db_password }}"
 DATABASE_NAME="{{ db_name }}"

